### DISCUSSION TITLE
- Final Project - EDA of NIFTY50 Dataset

### DISCUSSION THREAD

**Discussion by giscus** [DACSS/DACSS_601_August2022_v2#256] [Announcements]
# - Final Project - EDA of NIFTY50 Dataset

https://dacss.github.io/DACSS_601_August2022_v2/posts/FinalProject_AdithyaParupudi.html

**Comment by LarriMill** [comment]
Hi Adithya, nice job overall! In future assignments I encourage you to do a bit more story telling around the visuals that you use. Point your reader to what you want them to see, and have more descriptive labels. You did a great job of clearly describing your research questions.

**Comment by LarriMill** [comment]
Hi Yakub, you do a really great job with contextualizing this data and your research questions. You also make it easy for your readers to understand what they are looking at with your descriptions of your graphs and code. I would like to see more of a variety in the graphs that you created, and also -- correlation is not the same as causation!!!
